cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaMirrorMakerCluster.java

```diff
@@ -0,0 +1,445 @@
+
+    protected static final String ENV_VAR_KAFKA_MIRRORMAKER_BOOTSTRAP_SERVERS_PRODUCER = "KAFKA_MIRRORMAKER_BOOTSTRAP_SERVERS_PRODUCER";
+    protected static final String ENV_VAR_KAFKA_MIRRORMAKER_TRUSTED_CERTS_PRODUCER = "KAFKA_MIRRORMAKER_TRUSTED_CERTS_PRODUCER";
+    protected static final String ENV_VAR_KAFKA_MIRRORMAKER_TLS_AUTH_CERT_PRODUCER = "KAFKA_MIRRORMAKER_TLS_AUTH_CERT_PRODUCER";
+    protected static final String ENV_VAR_KAFKA_MIRRORMAKER_TLS_AUTH_KEY_PRODUCER = "KAFKA_MIRRORMAKER_TLS_AUTH_KEY_PRODUCER";
+    protected static final String ENV_VAR_KAFKA_MIRRORMAKER_SASL_PASSWORD_FILE_PRODUCER = "KAFKA_MIRRORMAKER_SASL_PASSWORD_FILE_PRODUCER";
+    protected static final String ENV_VAR_KAFKA_MIRRORMAKER_SASL_USERNAME_PRODUCER = "KAFKA_MIRRORMAKER_SASL_USERNAME_PRODUCER";
+
+    protected static final String ENV_VAR_KAFKA_MIRRORMAKER_WHITELIST = "KAFKA_MIRRORMAKER_WHITELIST";
+    protected static final String ENV_VAR_KAFKA_MIRRORMAKER_NUMSTREAMS = "KAFKA_MIRRORMAKER_NUMSTREAMS";
+
+    protected String whitelist;
+
+    protected KafkaMirrorMakerClientSpec producer;
+    protected CertAndKeySecretSource producerTlsAuthCertAndKey;
+    private String producerUsername;
+    private PasswordSecretSource producerPasswordSecret;
+    protected KafkaMirrorMakerConsumerSpec consumer;
+    protected CertAndKeySecretSource consumerTlsAuthCertAndKey;
+    private String consumerUsername;
+    private PasswordSecretSource consumerPasswordSecret;
+
+    /**
+     * Constructor
+     *
+     * @param namespace Kubernetes/OpenShift namespace where Kafka Mirror Maker cluster resources are going to be created
+     * @param cluster   overall cluster name
+     */
+    protected KafkaMirrorMakerCluster(String namespace, String cluster, Labels labels) {
+        super(namespace, cluster, labels);
+        this.name = kafkaMirrorMakerClusterName(cluster);
+        this.serviceName = serviceName(cluster);
+        this.validLoggerFields = getDefaultLogConfig();
+        this.ancillaryConfigName = logAndMetricsConfigName(cluster);
+        this.image = KafkaMirrorMakerSpec.DEFAULT_IMAGE;
+        this.replicas = DEFAULT_REPLICAS;
+        this.readinessPath = "/";
+        this.readinessTimeout = DEFAULT_HEALTHCHECK_TIMEOUT;
+        this.readinessInitialDelay = DEFAULT_HEALTHCHECK_DELAY;
+        this.livenessPath = "/";
+        this.livenessTimeout = DEFAULT_HEALTHCHECK_TIMEOUT;
+        this.livenessInitialDelay = DEFAULT_HEALTHCHECK_DELAY;
+        this.isMetricsEnabled = DEFAULT_KAFKA_MIRRORMAKER_METRICS_ENABLED;
+
+        this.mountPath = "/var/lib/kafka";
+        this.logAndMetricsConfigVolumeName = "kafka-metrics-and-logging";
+        this.logAndMetricsConfigMountPath = "/opt/kafka/custom-config/";
+    }
+
+    public static String kafkaMirrorMakerClusterName(String cluster) {
+        return cluster + KafkaMirrorMakerCluster.NAME_SUFFIX;
+    }
+
+    public static String serviceName(String cluster) {
```

Comment:
Are we using a `Service` for the Kafka Mirror Maker? Maybe we don't need this method.